src/grasshopper/lib/util/utils.py

```diff
@@ -20,13 +20,8 @@ def custom_trend(trend_name: str, extra_tag_keys=[]):
 
     def calc_time_delta_and_report_metric(func):
         def wrapper(journey_object, *args, **kwargs):
-            tags = {}
             try:
                 environment = journey_object.environment
-                host = environment.host
-                test_parameters = journey_object.scenario_args
-                if hasattr(journey_object.environment, "extra_context"):
```

Comment:
why are we getting rid of this line? do we not want to allow extra tags passed in this way anymore?

Reply:
it's all being passed in via self.tags now. All that logic can be done with just that local var, kinda like how its being done in that workflow run journey